etc/db_schema_whitelist.json

```diff
@@ -0,0 +1,49 @@
+{
+    "quote": {
+        "column": {
+            "alma_bnpl_eligibility": true,
+            "alma_cart_initiated_status": true
+        }
+    },
+    "quote_item": {
+        "column": {
+            "alma_insurance": true
+        }
+    },
+    "sales_order_item": {
+        "column": {
+            "alma_insurance": true
+        }
+    },
+    "alma_insurance_subscription": {
```

Comment:
This array doesn't need to be removed ?

Reply:
Whitelist file is mandatory to drop column and table at upgrade